application.py

```diff
@@ -5,6 +5,7 @@
 from dotenv import load_dotenv
 
 from flask import Flask
+from flask_cors import CORS
```

Comment:
getting a `ModuleNotFoundError: No module named 'flask_cors'` - did you install locally?

Reply:
oh, yes, I must have. I'll add it to requirements